custom_components/coway/sensor.py

```diff
@@ -37,18 +34,29 @@ async def async_setup_entry(
     sensors = []
 
     for purifier_id, purifier_data in coordinator.data.purifiers.items():
-            _LOGGER.debug(f'PURIFIER DATA for {purifier_id}: {purifier_data}\n')
+            LOGGER.debug(f'PURIFIER DATA for {purifier_id}: {purifier_data}\n')
             if purifier_data.mcu_version is None:
                 sensors.append(AirQualityIndex(coordinator, purifier_id))
+
             sensors.extend((
                 PreFilter(coordinator, purifier_id),
                 MAX2Filter(coordinator, purifier_id),
-                ParticulateMatter10(coordinator, purifier_id),
-                ParticulateMatter25(coordinator, purifier_id),
                 TimerRemaining(coordinator, purifier_id),
                 IndoorAQ(coordinator, purifier_id),
             ))
 
+            product_name = purifier_data.device_attr['product_name']
+            match purifier_data.device_attr['product_name']:
```

Comment:
Use the product_name variable here since you've already defined it.

Reply:
woops meant to delete the first line